components/currentAgenda.tsx

```diff
@@ -49,6 +49,7 @@ export const CurrentAgenda = ({
           />
         </StyledFeedbackActions>
       )}
+      <span></span>
```

Comment:
Hey that's what I was going to try if I could repro locally. Did it work?

Reply:
Yep, that did the trick. I suspect it's a race condition of some sort, with next/react trying to optimise something somewhere, resulting in the next element after that conditional block sometimes copping the wrong class when rendered.